lessons/beginners-en/circular-imports/index.md

```diff
@@ -1,35 +1,34 @@
 ## Circular imports
 
-In your homework about 1D tic-tac-toe you will have them in few modules.
-It will look like this:
-(Arrows between modules are showing the imports.)
+In the 1D tic-tac-toe game for your homework, you probably wrote several modules.
```

Comment:
They will write that homework after this lesson. Shouldn't this be in future tense? And the first task is to split one module into 4 modules. Homework in Czech https://pyladies.cz/v1/s005-modules/handout/handout5.pdf

Reply:
Ah, okay! Then "In the 1D tic-tac-toe game for your homework, you will probably write several modules."